### DISCUSSION TITLE
System.Net.Sockets.SocketException (10060)

### DISCUSSION THREAD

**Discussion by sanlukuangji** [Tyrrrz/DiscordChatExporter#781] [Q&A]
When I enter my token, the software could not show the guilds and the channels and after a period of time, an error appears: **System.Net.Sockets.SocketException (10060): A connection attempt failed because the connected party did not properly respond after a period of time**
How could I fix it?

**Comment by Tyrrrz** [comment]
Sounds like a generic timeout error. If it doesn't fix itself after a while, check your network/firewall settings to make sure nothing is preventing DCE from making HTTP requests.
